docker/recipe/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM alpine:3.9 AS builder
+
+WORKDIR /nmodl/src
+
+RUN apk add --update build-base gcc g++ make cmake flex flex-dev bison git python3-dev
+
+RUN pip3 install --trusted-host pypi.python.org jinja2 pyyaml pytest sympy
+
+
+RUN git clone --recursive https://github.com/BlueBrain/nmodl.git && \
+    cd nmodl && \
+    git checkout 0.2
```

Comment:
If someone rebuilt from Dockerfile, I would prefer to build master. what you think?

Reply:
yes, probably the right way would be to build from master. Once I understand how the automatic building on dockerhub works, I will fix this accordingly

Reply:
I mean, why can't we remove `checkout 0.2` from here?

Reply:
I suggest to make the version configurable at `docker build` time.

In Dockerfile:
```
ARG NMODL_VERSION=master
[...]
git checkout ${NMODL_VERSION}
```

Then you can specify another value when building the image: `docker build --build-arg NMODL_VERSION=0.2`

Reply:
that's also good idea. I will update that.